components/Icon.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+const icons = {
+    close: '\\e903',
+    hamburger: '\\e90b',
+};
+
+const Icon = styled.span`
+    font-family: 'icomoon' !important;
+    speak: none;
+    font-style: normal;
+    font-weight: normal;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```

Comment:
we can use the IconContainer for each Icon, just pass the iconCode to the container.
`const IconContainer = styled.span`
    font-family: 'icomoon' !important;
    &:before {
         content: "${(props) => icons[props.name]}";
     }
`;`